src/main/java/eu/enmeshed/model/relationships/Relationship.java

```diff
@@ -17,18 +17,13 @@
 @Setter
 @Builder
 public class Relationship implements WebhookData {
-
   private String id;
-
   private RelationshipTemplate template;
-
   private Status status;
-
   private String peer;
-
   private IdentityInfo peerIdentity;
```

Comment:
you are missing the optional field peerDeletionInfo:

```
export enum PeerDeletionStatus {
    ToBeDeleted = "ToBeDeleted",
    Deleted = "Deleted"
}

export interface PeerDeletionInfo {
    deletionStatus: PeerDeletionStatus;
}
```

Reply:
Done